torch/export/exported_program.py

```diff
@@ -600,11 +621,6 @@ def _decompose_exported_program(
         _is_executorch=False,
     )
 
```

Comment:
This isn't needed at this point, constants/non-persistent buffers won't be present as getattr nodes.